index.html

```diff
@@ -14,18 +14,23 @@
         <nav class="nav-list">
             
             <a class="logo" href="#">My Logo</a>
```

Comment:
[OPTIONAL]

- I will recommend you replace this logo with your actual name, thereby personalizing your application 👍🏽 
![image](https://user-images.githubusercontent.com/63473584/213544574-9997b4cc-306c-4583-9000-7b48b30277d2.png)